dashboard-ui/src/pages/home.tsx

```diff
@@ -50,6 +50,15 @@ import { Workload, allWorkloads, iconMap, labelsPMap } from '@/lib/workload';
 const basename = getBasename();
 
 const defaultKubeContext = appConfig.environment === 'cluster' ? '' : undefined;
+const defaultSidebarItems: Map<Workload, undefined> = new Map([
```

Comment:
It looks like you're using`defaultSidebarItems` to list the workloads in the sidebar while waiting for the counter data to finish loading. Instead of creating a new map to use as a placeholder, you can use `allWorkloads` to list all the workloads and then key into a map. You can also take advantage of the hook's `loading` variable. For example something like this should work:

```javascript
allWorkloads.map((workload) => (
  <div>
    <div>{labelsPMap[workload]}</div>
    {loading && (<div>count: {counter.get(workload) ?? 0}</div>)}
  </div>
))
```